.github/workflows/deploy-app-prod.yml

```
@@ -0,0 +1,42 @@
+name: Deploy an app to production
+
+on:
+  workflow_dispatch:
+    inputs:
+      app-name:
+        type: choice
+        description: Which app to deploy
+        options: 
+        - sage-account-web
```

Unfortunately we haven't been consistent with our app names
```suggestion
        - SageAccountWeb
```